front/app/components/IdeaCards/index.tsx

```diff
@@ -38,19 +36,6 @@ const Wrapper = ({
   </Container>
 );
 
-type IdeaCardsWithFiltersSidebarProps = Omit<WrapperProps, 'children'> &
-  WithSidebarProps;
-
-export const IdeaCardsWithFiltersSidebar = ({
-  className,
-  invisibleTitleMessage,
-  ...ideaCardsProps
-}: IdeaCardsWithFiltersSidebarProps) => (
-  <Wrapper className={className} invisibleTitleMessage={invisibleTitleMessage}>
-    <IdeasWithFiltersSidebar {...ideaCardsProps} />
-  </Wrapper>
-);
-
```

Comment:
Removed the wrapper for the `IdeasWithFiltersSidebar`, but not for the `WithoutFiltersSidebar`, since there is no other title in the `WithoutFiltersSidebar`.